km_api/know_me/models.py

```diff
@@ -110,6 +130,32 @@ def has_object_write_permission(self, request):
         return self.profile.user == request.user
 
 
+class KMUser(mixins.IsAuthenticatedMixin, models.Model):
+    """
+    A KMUser tracks information associated with each user of the Know Me app.
+
+    Attributes:
+        user:
+            The user who owns this profile.
+        image:
+            The user's main image.
+        quote (str) :
+            A quote from the user.
+    """
+    image = models.ImageField(
+        blank=True,
+        max_length=255,
+        null=True,
+        upload_to=get_km_user_image_upload_path,
+        verbose_name=_('image'))
+    quote = models.TextField(blank=True, null=True, verbose_name=_('quote'))
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name='km_user',
+        verbose_name=_('user'))
+
+
```

Comment:
If you could add a `__str__` method here that would be great. It should look roughly like:

```python
def __str__(self):
    return self.user.get_short_name()
```